pages/dmp_supporting_info/support_00_dmp_requirements.md

```diff
@@ -33,11 +33,15 @@ DMP requirements are commonly part of Open Science policies or guidelines for re
 * [UiT Principles and guidelines for managment for research data](https://en.uit.no/research/research-dataportal/art?p_document_id=729171#mod_729184 "2024-07-17")
 > The researcher must set up a Data Management Plan at an early phase of the project and preferably within six months after start-up.
 
-## National strategy on access and sharing of research data
+## National strategy on access to and sharing of research data
 * [Norwegian Ministry of Education and Research. (2017). National strategy on access to and sharing of research data](https://www.regjeringen.no/en/dokumenter/national-strategy-on-access-to-and-sharing-of-research-data/id2582412/ "2024-09-09")
+> Basic principles:
 > * Research data must be as open as possible, as closed as necessary
 > * Research data should be managed and curated to take full advantage of their potential
 > * Decisions concerning archiving and data management must be taken within the research community
+> Expectation:
+> * The Government expects the research institutions to develop procedures for (i) approving data management plans and (ii) determining whether a given research project is of a
```

Comment:
Er det noe som vi ønsker å fremheve for forskere, særlig siden ikke alle institusjoner er rigget for å ta imot og vurdere planer?

Reply:
Det siterte dokumentet er fra 2018. Under betraktning av implementerings statusen bør regeringen revidere disse ambisjonene og/eller implementere dem
Dette er også et strategidokument, ikke en retningslinje...

Reply:
Jeg syns det kan være et poeng å gjengi denne forventningen fra myndighetene. Det er et spørsmål vi får til tider, om hvem som skal godkjenne DMPen. Hvis dette er urealistisk så bør vel departementet revidere sitt strategidokument. (Og vi sier jo i overskriften at det er et strategidokument.)